artacle-swagger.yaml

```diff
@@ -314,6 +314,281 @@ paths:
+                properties:
+                  data:
+                    $ref: '#/components/schemas/CollectionTrait'
+
+        '401':
+          $ref: "#/components/responses/UnauthorizedError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '404':
+          $ref: "#/components/responses/NotFoundError"
+        '429':
+          $ref: "#/components/responses/TooManyRequestsError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+          
+#  Collection Trait TTF
+  /collections/{cid}/trait/ttf/v1:
+    get:
+      tags:
+        - Traits
+      summary: Get particular trait trait-to-floor information. all prices and volumes by default are provided in the native collection currency (i.e. ETH for Ethereum)
+      description: Returns particular collection trait trait-to-floor information
+      parameters:
+        - name: cid
+          in: path
+          required: true
+          description: Artacle collection ID as returned by the /collections/ endpoint
+          schema:
+            type: integer
+            example: 78
+        - name: apiKey
+          in: query
+          required: true
+          description: API access key
+          schema:
+            type: string
+        - name: traitName
+          in: query
+          required: true
+          description: Query only traits with certain name
+          schema:
+            type: string
+            example: Color
+        - name: traitValue
+          in: query
+          required: true
+          description: Query only traits with certain value
+          schema:
+            type: string
+            example: Color
+        - name: startTimestamp
+          in: query
+          required: false
+          description: Unix timestamp. Indicates start time of collected samples. Method fetches everything till endTimestamp if not specified.
+          schema:
+            type: number
+            format: uint64
+        - name: endTimestamp
+          in: query
+          required: false
+          description: Unix timestamp. Indicates end time of collected samples. Set to now by default.
+          schema:
+            type: number
+            format: uint64
+        - name: includeSales
+          in: query
+          required: false
+          schema:
+            type: number
+            enum:
+              - 0
+              - 1
+            default: 0
+        - name: offset
+          in: query
+          required: false
+          description: Results pagination
+          schema:
+            type: integer
+            default: 0
+            example: 0
+        - name: limit
+          in: query
+          required: false
+          description: Limit elements to fetch
+          schema:
+            type: integer
+            example: 50
+            default: 50
+            maximum: 1000
+      responses:
+        '200':
+          description: Success, returns specific trait TTF information
```

Comment:
Suggesting to add a note 'within specified time range' - it is an important distinction of this method